pkg/abstract/slices.go

```diff
@@ -25,3 +25,23 @@ func SliceDelete[Elt any](slice *[]Elt, index int) {
 	}
 	*slice = (*slice)[:lastIndex]
 }
+
+// SliceCopy copies a given slice into new storage
+func SliceCopy[Elt any](input []Elt) []Elt {
```

Comment:
Isn't this actual a CreateFrom rather than copy ?

Reply:
In plain English, "copy" connotes making a new thing. When I copy a sculpture, I make a new one that looks like an existing one. It is true that Unix & C introduced things like "memcpy", which writes the contents of one block of memory into another pre-existing block of memory, and Go has a built-in "copy" that does something similar. But SliceCopy here is unlike those, in that it does not take two slices as input, it takes just one and returns a new slice (new memory). So the English word "copy" is quite appropriate here.